skimage/segmentation/_watershed.py

```diff
@@ -121,6 +121,9 @@ def watershed(image, markers=None, connectivity=1, offset=None, mask=None,
     watershed_line : bool, optional
         If watershed_line is True, a one-pixel wide line separates the regions
         obtained by the watershed algorithm. The line has the label 0.
+        Note that the method used for adding the watershed line expects that
```

Comment:
```suggestion
        Note that the method used for adding this line expects that
```
(follows the previous sentences more naturally)

Reply:
Updated. Thank you!